### PR TITLE
[feature/51622] Adjusted what is new flow and texts

### PR DESCRIPTION
**Pull request by jonasofrich**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19/Locales/en.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
   "WELCOME_PAGE_WHATS_NEW_BULLET_ONE": "Smittestopp now works in EU and allows you to receive a message if you encounter infected EU citizens with a COVID-19 app in Norway. ",
   "WELCOME_PAGE_WHATS_NEW_BULLET_TWO": "Smittestopp now helps to stop the spread of COVID-19 in EU countries. You can give your consent to share your status so European COVID-19 app users can receive a message if you have been nearby – no one will get to know who you are or where you are from.",
   "WELCOME_PAGE_WHATS_NEW_BUTTON": "Next",
-  "WELCOME_PAGE_WHATS_NEW_FOOTER": "Because of the new features you have to give your consent again on the next page.",
+  "WELCOME_PAGE_WHATS_NEW_FOOTER": "If you would like to withdraw your consent because of these changes, you can do this in the \"Consent and privacy\" page in the main menu.",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/en.json:114]: Instead of "main menu" please call it "Menu"

File: NDB.Covid19/NDB.Covid19/Locales/nb.json [hunk 2]
```diff
@@ -111,7 +111,7 @@
   "WELCOME_PAGE_WHATS_NEW_BULLET_ONE": "Smittestopp fungerer nå i EU, og lar deg motta en melding hvis du møter smittede EU-borgere med en covid-19-app i Norge.",
   "WELCOME_PAGE_WHATS_NEW_BULLET_TWO": "Smittestopp er nå med på å stoppe spredningen av covid-19 i EU-land. Du kan, hvis du blir smittet, gi ditt samtykke til å dele statusen din slik at europeiske covid-19-app brukere kan motta en melding hvis dere har vært i nærheten av hverandre. Ingen vil få vite hvem du er eller hvor du er fra.",
   "WELCOME_PAGE_WHATS_NEW_BUTTON": "Neste",
-  "WELCOME_PAGE_WHATS_NEW_FOOTER": "På grunn av de nye funksjonene må du gi ditt samtykke igjen på neste side.",
+  "WELCOME_PAGE_WHATS_NEW_FOOTER": "Hvis du vil trekke tilbake ditt samtykke på grunn av disse endringene, kan dette gjøres på \"Samtykke og personvern\"-siden i hovedmenyen.",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/nb.json:114]: as above

**Review by Frederik-S-N-dk — APPROVED** [review]

**Review by Frederik-S-N-dk — CHANGES_REQUESTED** [review]
see previous review from me

File: NDB.Covid19/NDB.Covid19/Locales/en.json (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/Locales/en.json]: done

File: NDB.Covid19/NDB.Covid19/Locales/nb.json (hunk 2 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/Locales/nb.json]: done

**Review by Frederik-S-N-dk — APPROVED** [review]

**Pull request merged**